homeassistant/components/light/osramlightify.py

```diff
@@ -198,6 +196,8 @@ def turn_on(self, **kwargs):
             self._luminary.set_luminance(
                 int(self._brightness / 2.55),
                 transition)
+        else:
+            self._luminary.set_onoff(1)
```

Comment:
I checked this point. When the light is off the Home Assistant UI for lightify light component does not expose rgb and temperature controls (only brightness and on/off switch) so I am not able to test if it could introduce a problem.
Anyway, with the Home Assistant UI currently, this is not a use case.